.github/ISSUE_TEMPLATE/installation_issue_template.yaml

```diff
@@ -0,0 +1,10 @@
+name: Installation Issues
```

Comment:
I don't think we have to single installation out. It's a relatively simple template repo. Issues can be aggregated under the generic "BUG" label. Easier to triage there.